src/components/Thread.vue

```diff
@@ -50,26 +51,23 @@ export default {
 			message: undefined,
 			errorMessage: '',
 			error: undefined,
+			thread: [],
```

Comment:
:-1: this makes the thread non-reactive

Reply:
new version proposed. Shall I create a new PR?